sys-libs/libatomic_ops/libatomic_ops-7.6.2.recipe

```diff
@@ -0,0 +1,65 @@
+SUMMARY="An atomic memory update operations portable implementation"
+DESCRIPTION="Semi-portable access to hardware-provided atomic memory update operations on a number architectures."
+HOMEPAGE="https://github.com/ivmai/libatomic_ops"
+COPYRIGHT="2003-2011 Hewlett-Packard Development Company, L.P."
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/ivmai/libatomic_ops/releases/download/v$portVersion/libatomic_ops-$portVersion.tar.gz"
+CHECKSUM_SHA256="219724edad3d580d4d37b22e1d7cb52f0006d282d26a9b8681b560a625142ee6"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	libatomic_ops$secondaryArchSuffix = $portVersion
+	lib:libatomic_ops$secondaryArchSuffix = 2.1.1 compat >= 1
+	lib:libatomic_ops_gpl$secondaryArchSuffix = 2.1.2 compat >= 1
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libatomic_ops$secondaryArchSuffix_devel = $portVersion
+	devel:libatomic_ops$secondaryArchSuffix = 2.1.1 compat >= 1
+	devel:libatomic_ops_gpl$secondaryArchSuffix = 2.1.2 compat >= 1
+	"
+REQUIRES_devel="
+	libatomic_ops$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:gawk
+	cmd:gcc$secondaryArchSuffix
+	cmd:grep
+	cmd:ld$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:sed
+	"
+
+BUILD()
+{
+	runConfigure ./configure --disable-dependency-tracking \
+		--enable-shared \
+		--with-gnu-ld
+
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+
+	fixPkgconfig
+
+	prepareInstalledDevelLibs \
+			libatomic_ops libatomic_ops_gpl
+
+	# devel package
+	packageEntries devel \
+			$developDir $dataDir
```

Comment:
```diff
	packageEntries devel \
-			$developDir $dataDir
+		$developDir
```
With `libatomic_ops-7.4.4.recipe` we do get a `$dataDir`, with these contents:
```
data/libatomic_ops/COPYING
data/libatomic_ops/LICENSING.txt
data/libatomic_ops/README.md
data/libatomic_ops/README.txt
data/libatomic_ops/README_malloc.txt
data/libatomic_ops/README_stack.txt
data/libatomic_ops/README_win32.txt
```
But with `libatomic_ops-7.6.2.recipe` the build does not install these.

Indeed, one of the changes upstream made to the main `Makefile.am` is:
```diff
---libatomic_ops-7.4.4/Makefile.am
+++libatomic_ops-7.6.2/Makefile.am
@@ -6,8 +6,19 @@
 pkgconfig_DATA = pkgconfig/atomic_ops.pc
 noinst_DATA = pkgconfig/atomic_ops-uninstalled.pc
 
-dist_pkgdata_DATA = COPYING README.md
+if ENABLE_DOCS
+dist_doc_DATA = COPYING README.md
+endif
 
 EXTRA_DIST = autogen.sh
```

The good news is that the `README.md` file, with 4 other `README_*.txt` are now in `$docDir`.
So all you need to do is drop the `$dataDir` from line 64, as I suggested in the diff on the top of this message.

Reply:
Hmm, @ivmai, I've just realized that **_you_** are the upstream author! This is great!